### PR TITLE
TECH-3693 ckeditor autosave

### PR DESCRIPTION
**Pull request by greglinch**
[TECH-3693 implement CKeditor-specific AutoSave plugin](https://industrydive.atlassian.net/browse/TECH-3693)

More info on the [Confluence page](https://industrydive.atlassian.net/wiki/spaces/TECH/pages/428934055/CKeditor+autosave+plugin)
 
### In a nutshell
Reporters and editors sometimes lose their progress to browser or computer crashes. We currently have no way to recover lost work, so this update aims to remedy that for all CKeditor fields. 

### Non-technical Context
Neither Django nor CKeditor have built-in autosave tools, so we are implementing this to address the lack of autosave for CKeditor fields. Another tool — The Atlantic's django-autosave — was tested and shelved because it didn't work with CKeditor fields. 

Although this update only addresses CKeditor fields, these fields are the most important ones to recover due to their potential length. Specifically, it's possible to lose a significant amount of time if custom teaser, body or note contents are lost. On the other hand, losing the other shorter fields is annoying, but won't set someone back significantly in terms of time.
 
### Technical Context
This update replaces the previously unused CKeditor [autosave plugin](https://ckeditor.com/cke4/addon/autosave) we had installed with the latest version, along with the [notification plugin](https://ckeditor.com/cke4/addon/notification) dependency. 

Based on Davide's input, it also sets some custom configurations for how the autosave should behave: 

- _delay_: sets the frequency of autosaves
- _NotOlderThen_: sets the length of time before an autosave expires

Additional configuration options are [available here](https://github.com/w8tcha/CKEditor-AutoSave-Plugin#to-configure-the-plugin-the-following-options-are-available).

NOTE: The `reqs.txt` file in divesite CMS repo will need to be updated with the new version number.

### Reproduction Steps
n/a -- not a bug

### Definition of Done
This only covers CKeditor fields and not all newspost fields.

### QA Checklist
*Specific*
- [ ] Add a new newspost or edit an existing newspost on wetsuit (e.g. [this one](http://wetsuit.constructiondive.com/admin/news/newspost/506741/))
- [ ] For a new newspost add a title, section and save
- [ ] Adding new text to the `custom_teaser`, `body` and `note` fields
- [ ] Return to the [News Post list view](http://wetsuit.constructiondive.com/admin/news/newspost/) and click "Leave" when prompted
- [ ] Open that post again
- [ ] Click "Ok" on the prompt to view the autosave diff
- [ ] Click "Yes" or "No"
- [ ] Confirm that the autosaved text appears as expected

*General*
- [ ] Devise edge cases to test the bug fix or feature being merged
- [ ] Make sure the code is clean and understandable
    - [ ] Ask for inline comments on unclear sections
- [ ] Ensure code is not overly inefficient
- [ ] Ensure documentation is understandable and accurate, including:
    - [ ] Code comments and doc strings
    - [ ] Public docmentation/how-to guides
    - [ ] Technical documentation
 - [ ] Set deployment notes on Jira ticket

### CODE REVIEW

File: ckeditor/static/ckeditor/ckeditor/config.js [hunk 1]
```diff
@@ -7,4 +7,5 @@ CKEDITOR.editorConfig = function( config ) {
 	// Define changes to default configuration here. For example:
 	// config.language = 'fr';
 	// config.uiColor = '#AADC6E';
+	config.extraPlugins = 'autosave';
```
Comment (MiriamSexton) [inline ckeditor/static/ckeditor/ckeditor/config.js]: I don't think we want to add this here. We don't *always* want autosave, just on instance we specify

File: ckeditor/static/ckeditor/ckeditor/config.js [hunk 2]
```diff
@@ -4,7 +4,18 @@
...
 
 CKEDITOR.editorConfig = function( config ) {
-	// Define changes to default configuration here. For example:
-	// config.language = 'fr';
-	// config.uiColor = '#AADC6E';
+    // Define changes to default configuration here. For example:
+    // config.language = 'fr';
+    // config.uiColor = '#AADC6E';
+    config.extraPlugins = 'notification';
+    config.extraPlugins = 'autosave';
```
Comment (MiriamSexton) [inline ckeditor/static/ckeditor/ckeditor/config.js]: I don't think we want to add this here. We don't always want autosave, just on instance we specify

File: ckeditor/static/ckeditor/ckeditor/config.js [hunk 3]
```diff
@@ -4,7 +4,18 @@
...
+    config.autosave = {
```
Comment (MiriamSexton) [inline ckeditor/static/ckeditor/ckeditor/config.js]: see above, this is related
